### DISCUSSION TITLE
Output Format: GitHub

### DISCUSSION THREAD

**Discussion by alexanderkroneis** [rectorphp/rector#8573] [Ideas]
Hi,

I‘d love to see a new output format using GitHub Annotations like PHPStan is doing it.

**Comment by TomasVotruba** [comment]
Hi, 

we're not planning this feature, as PHPStan is report only and decorating PR makes sense there for collecting feedback.
Rector changes the code directly.

Still, you can implement your own output formatter with this feature

**Reply by scollovati** [reply]
If I am not wrong, I spotted this option in rector: `--output-format=github`
